test/functionalTest/cases/3068_non_legacy_notifications/legacy_forwarding_true_false_default_with_accumulator.test

```diff
@@ -0,0 +1,112 @@
+# Copyright 2019 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
We should provide in positive, I mean, instead of "non legacy notifications" (notifications here is wrong, by the way as this is not related with subscriptinos) we should use something like "ngsiv2 forwarding".

Not only for directory and .test file names, but also for step descriptions and echo messages inside (if applicable).



(In front of me from the very begingin, but I have just realized about this, sorry)

Reply:
Yeah, I saw it too.
A forwarding request is so similar to a notification ...
I make this mistake all the time :(

Reply:
Fixed in ad0ca86a2